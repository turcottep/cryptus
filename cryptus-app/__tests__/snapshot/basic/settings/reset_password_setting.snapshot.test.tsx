import React from "react";
import renderer from "react-test-renderer";
import "@testing-library/jest-dom/extend-expect";

import ResetPasswordSetting from "../../../../components/basic/settings/reset_password_setting/reset_password_setting";

describe("<ResetPasswordSetting />", () => {
  it("should display properly", async () => {
    const tree = renderer.create(<ResetPasswordSetting />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
