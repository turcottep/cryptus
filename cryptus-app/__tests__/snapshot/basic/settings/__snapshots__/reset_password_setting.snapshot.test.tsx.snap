// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ResetPasswordSetting /> should display properly 1`] = `
<div>
  <div>
    Reset Password
    <div>
      <img
        src="/icons/back_icon.png"
      />
    </div>
  </div>
</div>
`;
